### PR TITLE
Classifier

### PR DESCRIPTION
**Pull request by Sidu28**
Binary classifier implemented using a basic neural net.  For now, the dimensions are hardcoded and the predict function generates a random X value to predict the label for.  Once we define our features better, we make these dynamic.

### CODE REVIEW

File: classifer.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+
+
+import tensorflow as tf
+from tensorflow import keras
+
+from sklearn.datasets import make_blobs
+from sklearn.model_selection import train_test_split
+
+class BinaryClassifier(object):
```
Comment (mananshah99) [inline classifer.py:14]: Would be good to rename this flie to `binary_classifier.py` so we can add other classifiers as necessary.

**Review by mananshah99 — COMMENTED** [review]
LGTM aside from one minor change about renaming classifier.py, which I assume will also affect README.md.

File: classifer.py (hunk 1 above)
Comment (Sidu28) [reply classifer.py:14]: Done!
